src/modules/75-cf/components/CreateEnvironmentDialog/__tests__/EnvironmentDialog.test.tsx

```diff
@@ -110,7 +187,8 @@ describe('EnvironmentDialog', () => {
     })
 
     test('it should not show error message if Environment name contains an ASCII character', async () => {
-      renderComponent()
+      const onCreateEnv = jest.fn()
+      renderComponent({ onCreate: onCreateEnv })
```

Comment:
Looks like this isn't asserted against. Might as well omit and let the default take over
```suggestion
      renderComponent()
```

Reply:
Probably left over from previous coding attempts 😄